wled00/set.cpp

```diff
@@ -771,7 +771,7 @@ void handleSettingsSet(AsyncWebServerRequest *request, byte subPage)
 //HTTP API request parser
 bool handleSet(AsyncWebServerRequest *request, const String& req, bool apply)
 {
-  if (!(req.indexOf("win") >= 0)) return false;
+  if (!(req.indexOf(F("win")) >= 0)) return false;
```

Comment:
This one is unnecessary as "win" is also used elsewhere and AFAIK same strings only occupy space once if not stored in PROGMEM.

Reply:
It's true that string literals can be deduplicated by the linker, while PROGMEM arrays have to be deduplicated by the programmer.  Unfortunately, our ESP8266 RAM is basically filled with them.  I'll see if I can get some metrics, but we're looking at many kb in total.  Browsing the RAM layout, it's just page after page after page of static strings, almost all of which could be in PROGMEM if the surrounding code supported it.

I've got a note in my todo to look at this from a more systematic perspective - I want to prod C++11 user-defined literals and the linker scripts - but I don't think I'll get to that for a while.  For the forseeable future, about the only option we have is to go through literal by literal and move everything we can, one patch at a time.

In any event, I'll drop 4c49241 from the PR, and we can consider that process elsewhere.